torch/_torch_docs.py

```diff
@@ -4338,6 +4338,80 @@ def merge_dicts(*dicts):
     {out}
 """.format(**common_args))
 
+add_docstr(torch.xlogy,
+           r"""
+xlogy(self, other, *, out=None) -> Tensor
+
+Compute ``self * log(other)`` so that ``0 * log(0)`` returns zero.
+
+Similar to SciPy's `scipy.special.xlogy`.
+
+.. math::
+    \text{out}_{i} = \begin{cases}
+        \text{NaN} & \text{if } \text{other}_{i} = \text{NaN} \\
+        0 & \text{if } \text{self}_{i} = 0.0 \\
+        self_i * \log{(other_i)} & \text{otherwise}
+    \end{cases}
+
+""" + r"""
+
+.. note::
+    ``NaN`` in :attr:`other` is propagated even if value of :attr:`self` is 0.
+
+Args:
+    self (Tensor)
+    other (Number or Tensor)
+
+Keyword args:
+    {out}
+
+Example::
+
+    >>> x = torch.zeros(4,)
+    >>> y = torch.tensor([0, 1, float('inf'), float('nan')])
+    >>> torch.xlogy(x, y)
+    tensor([0., 0., 0., nan])
+    >>> x = torch.tensor([1, 2, 3])
+    >>> y = torch.tensor([3, 2, 1])
+    >>> torch.xlogy(x, y)
+    tensor([1.0986, 1.3863, 0.0000])
+    >>> torch.xlogy(x, 4)
+    tensor([1.3863, 2.7726, 4.1589])
+
+""".format(**common_args) + r"""
+
+.. function:: xlogy(self, other, *, out=None) -> Tensor
```

Comment:
First entry covers two variants,
x -> tensor, y -> float, tensor

Second entry covers,
x -> float, y -> tensor.

If we remove the second, then x -> float, y -> tensor won't be covered by doc. 

This is similar to `torch.pow`'s doc.
https://github.com/pytorch/pytorch/blob/a39398b9e5d528e4a6ca293f1703833932f0d9b2/torch/_torch_docs.py#L6403

Reply:
Can they not be merged to describe that either x or y can be a Number or Tensor? It's true that this may mislead users into believing they can pass both inputs as Numbers, but that seems OK.

Reply:
Cool. Will do that.